docs/General/Manuals/client-manual.rst

```diff
@@ -92,10 +92,44 @@ From upper-left to lower-right, the following user interface elements are availa
 Players List
 ^^^^^^^^^^^^
 
+The :guilabel:`Players list` table shows information about the configured players in the game. The information
+shown in Figure 2 is what a single-player game looks like. A LongTurn multi-player game will look very
+similar, except that all the player's aliases will be shown as setup by the game administrator.
+You can right-click on a player's row to configure details about the specific player:
+
+* :guilabel:`Observe` -- Allows you to connect to a running game and observe that player. This is useful
+  during LongTurn multi-player games when you want to connect and see what a player is doing, but you cannot
+  make any actual moves for the player. This works for LAN games as well. You can also use the server
```

Comment:
You can also use this to cheat on the AI :D

Reply:
While that is interesting, I'm not going to put it in the manual.